crates/bevy_gltf/src/loader.rs

```diff
@@ -309,6 +304,23 @@ async fn load_gltf<'a, 'b>(
                 }
             }
 
+            if let Some(vertex_attribute) = reader
```

Comment:
For future reviewers, this reordering is done to make sure that all the data has been added to the mesh before `bevy_mikktspace` is run.